### ISSUE TITLE
Framebuffer with different number of attachments

### ISSUE THREAD

**Issue by dati91** [szeged/webrender#85]
```
ERROR:: [DS] You cannot start a render pass using a framebuffer with a different number of attachments.
ERROR:: [DS] vkCmdBeginRenderPass(): RenderPasses incompatible between render pass w/ renderPass 0x6 and framebuffer w/ renderPass 0x7 Attachment 4294967295 is not compatible with 1: The first is unused while the second is not.. The spec valid usage text states 'renderPass must be compatible with the renderPass member of the VkFramebufferCreateInfo structure specified when creating framebuffer.' (https://www.khronos.org/registry/vulkan/specs/1.0/html/vkspec.html#VUID-VkRenderPassBeginInfo-renderPass-00904)
```

When we create our framebuffers from the backbuffer images, we always create them with depth. But there are pipelines with depth test turned off, which use these framebuffers.

- [ ] Create framebuffers with and without depth for backbuffer images

**Comment by kvark** [comment]
TL;DR: a render pass that uses depth is not compatible with one that doesn't
